src/main/java/duke/command/MyParser.java

```diff
@@ -0,0 +1,134 @@
+package duke.command;
+
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.Calendar;
+import java.util.Date;
+
+/**
+ * This class implements a parser that deals with making sense of user commands.
+ *
+ * CS2103T ip
+ * AY21/22 Semester 1
+ *
+ * @author Kishendran Vendar Kon (Group G05)
+ */
+public class MyParser {
```

Comment:
Maybe `Parser` will be a better name? It is more concise, and the `my` seems to have no special meaning.